sdk/eventhubs/test-resources.json

```diff
@@ -0,0 +1,351 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
Is this expected to be under `sdk/eventhubs`? Should this be under a specific module? For e.g. `sdk/eventhubs/azure-messaging-eventhubs`?

Reply:
This is right place based on current strategy as we have already done in storage also.